packages/ui/src/composites/wui-loading-spinner/index.ts

```diff
@@ -0,0 +1,28 @@
+import { html, LitElement } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+import '../../components/wui-icon'
+import { resetStyles } from '../../utils/ThemeUtil'
+import styles from './styles'
+
+@customElement('wui-loading-spinner')
+export class WuiLoadingSpinner extends LitElement {
+  public static styles = [resetStyles, styles]
+
+  // -- State & Properties -------------------------------- //
+
+  @property({ type: Boolean }) public loading = true
```

Comment:
To stick with common patterns, we can remove loading prop, this will simply remain as an active loading component at all times. Then in scaffold, we will determine when to render or not render the loaders. It is important to remove the from dom when they are not needed, not just set their opacity to 0. (we can animate from scaffold as well if we want)